easycv/models/backbones/vision_transformer.py

```diff
@@ -635,7 +595,7 @@ def forward(self, x):
 
 # todo: refactor vitdet and vit_transformer_dynamic
 @BACKBONES.register_module()
-class ViTDet(nn.Module):
```

Comment:
keep original name until we merge those vision transformers into one.  This vit is adapted with support for vitdet

Reply:
done